flip_fab.gemspec

```diff
@@ -12,8 +12,6 @@ Gem::Specification.new do |spec|
   spec.homepage      = 'https://github.com/simplybusiness/flip_fab'
   spec.license       = 'MIT'
 
-  spec.metadata['allowed_push_host'] = 'https://gemstash.simplybusiness.io/private'
-
```

Comment:
just out of curiosity. ATM push host is defined by the credentials that we're setting up in the pipeline step?

Reply:
It is using the default push host.  This is copied from twiget-ruby repo https://github.com/simplybusiness/twiglet-ruby